src/System.Threading.Tasks.Dataflow/src/Base/DataflowBlock.cs

```diff
@@ -17,7 +17,10 @@
 using System.Diagnostics.Contracts;
 using System.Runtime.CompilerServices;
 using System.Threading.Tasks.Dataflow.Internal;
+
+#if !THREADING
```

Comment:
"THREADING" is a fairly generic constant name.  How about "USE_INTERNAL_THREADING" or something like that?